micrometer-core/src/test/java/io/micrometer/core/instrument/binder/jvm/JvmCompilationMetricsTest.java

```diff
@@ -23,7 +23,7 @@
 import java.lang.management.ManagementFactory;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
-import static org.junit.Assume.assumeTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
```

Comment:
Thanks for catching this. I've pushed this fix to `master` now in 5876298bd66443ecefe4c064a9ea67f98026bf66. I'm confused how the build passed with this before... but nonetheless fixed now.

Reply:
thx for follow-up!